### ISSUE TITLE
Please update this plugin on poggit!

### ISSUE THREAD

**Issue by kevinishawesome** [Falkirks/MineReset#111]
This plugin on poggit isn’t compatible with new API, please update it there too! Will be appreciated!

**Comment by Falkirks** [comment]
@kevinishawesome I haven't been able to test it yet which is why it hasn't been promoted on poggit. I need to get my actual MCPE client to work to visually check the mines have been reset ok.

**Comment by pietro21x** [comment]
yes pls update error: 17:47:09] [Asynchronous Worker #0 thread/CRITICAL]: ErrorException: "Undefined offset: 0" (EXCEPTION) in "MineReset_dev-37.phar/src/falkirks/minereset/task/ResetTask" at line 58
[17:47:09] [Asynchronous Worker #0 thread/DEBUG]: #0 MineReset_dev-37.phar/src/falkirks/minereset/task/ResetTask(58): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string Undefined offset: 0, string phar:///root/1/ForceCraft/plugins/MineReset_dev-37.phar/src/falkirks/minereset/task/ResetTask.php, integer 58, array Array())
[17:47:09] [Asynchronous Worker #0 thread/DEBUG]: #1 src/pocketmine/scheduler/AsyncTask(72): falkirks\minereset\task\ResetTask->onRun()
